vivisect/vivbin.py

```diff
@@ -54,6 +56,9 @@ def main():
     level = e_common.LOG_LEVELS[vw.verbose]
     e_common.initLogging(logger, level=level)
 
+    if args.outfile:
+        vw.setMeta('StorageName', args.outfile)
```

Comment:
How does this impact viv file portability if the file that we loaded from doesn't match this persisted value?

Reply:
`loadWorkspace()` clobbers this value each time the workspace is loaded, with the new name.